### PR TITLE
[Fix] Fix GPU memory leak

### LINKED ISSUE #96: GPU memory unbalance problem
I copied the `dist_train.sh` from mmpose.
```sh
#!/usr/bin/env bash
# Copyright (c) OpenMMLab. All rights reserved.

CONFIG=$1
GPUS=$2
PORT=${PORT:-29500}

PYTHONPATH="$(dirname $0)/..":$PYTHONPATH \
python -m torch.distributed.launch --nproc_per_node=$GPUS --master_port=$PORT \
    $(dirname "$0")/train.py $CONFIG --launcher pytorch ${@:3}
```
With the following training command:
```
tools/dist_train.sh configs/hmr/resnet50_hmr_pw3d.py 2 --no-validate
```

When I have `samples_per_gpu=32`, and `workers_per_gpu=2`,  I found that the GPU memory is unbalanced.

image

### PR DESCRIPTION
**Pull request by leng-yue**
Resolves #96: Init camera and SMPl statically waste GPU memory and lead to GPU memory unbalance (all these models are loaded on GPU:0).

### CODE REVIEW

**Comment by caizhongang** [comment]
Hi @leng-yue , thank you very much for your contribution! However, this fix is already included in https://github.com/open-mmlab/mmhuman3d/pull/87/files#diff-fb40a8b8b1d1bbc4de3cd1fa9f83edbf166cd079f4401c1994933110262862d4 which will be merged soon.

**Comment by caizhongang** [comment]
Hi @leng-yue , thank you very much for your contribution, although this fix is included in an older PR (soon to merge), https://github.com/open-mmlab/mmhuman3d/pull/87/files#diff-fb40a8b8b1d1bbc4de3cd1fa9f83edbf166cd079f4401c1994933110262862d4, we consider it may be clearer to fix by your PR! Please fix the linting issue and the PR is good to go! Let us know if you face any difficulties.

**Comment by leng-yue** [comment]
Lint problems should be fixed now, waiting for Github Action.

**Comment by leng-yue** [comment]
#96

File: mmhuman3d/data/data_converters/gta_human.py [hunk 1]
```diff
@@ -29,38 +29,40 @@ class GTAHumanConverter(BaseConverter):
...
-            image_size=image_size,
-            principal_point=camera_center))
+    def __init__(self, *args, **kwargs):
+        super(GTAHumanConverter, self).__init__(*args, **kwargs)
+
+        focal_length = 1158.0337
+        camera_center = (960, 540)  # xy
+        image_size = (1080, 1920)  # (height, width)
+
+        device = torch.device(
```
Comment (caizhongang) [inline mmhuman3d/data/data_converters/gta_human.py:39]: Here `device` should be `self.device`.

**Review by caizhongang — APPROVED** [review]

**Pull request merged**